drivers/input/djoystick.c

```diff
@@ -181,12 +162,6 @@ static void djoy_enable(FAR struct djoy_upperhalf_s *priv)
   lower = priv->du_lower;
   DEBUGASSERT(lower);
 
-  /* This routine is called both task level and interrupt level, so
-   * interrupts must be disabled.
-   */
-
-  flags = enter_critical_section();
```

Comment:
Probably need to remove `irqstate_t flags;` as well

Reply:
Done.